projects/observability/src/shared/services/filter-builder/graphql-filter-builder.service.ts

```diff
@@ -37,10 +38,16 @@ export class GraphQlFilterBuilderService {
         new GraphQlFieldFilter(
           { key: filter.field, subpath: filter.subpath },
           toGraphQlOperator(filter.operator),
-          filter.value as GraphQlArgumentValue
+          this.transformValueIfOperatorIsContains(filter)
         )
     );
   }
+
+  private transformValueIfOperatorIsContains(filter: Filter | FieldFilter | TableFilter): GraphQlArgumentValue {
```

Comment:
nit: would rename to just something like "extractGraphQlFilterValue" or similar. Basically, the contains bit is internal to this function.

Reply:
good point, fixed